source/CC_RemoteTech/CC_RemoteTech.csproj

```diff
@@ -36,11 +37,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Kerbal Space Program\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
+      <HintPath>$(KspInstallationDir)\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
LinuxGuruGamer suggested a change to PKS which I think you might want to do, which is to add the references in bulk, like so:

```
    <Reference Include="$(KspInstallationDir)\KSP_x64_Data\Managed\Assembly*.dll">
      <Private>False</Private>
    </Reference>
    <Reference Include="$(KspInstallationDir)\KSP_x64_Data\Managed\UnityEngine*.dll">
      <Private>False</Private>
    </Reference>

```